app/src/main/java/com/duckduckgo/app/email/EmailInjector.kt

```diff
@@ -77,9 +77,9 @@ class EmailInjectorJs(
         url: String?
     ) {
         url?.let {
-            if (isDuckDuckGoUrl(url) || (isFeatureEnabled() && !autofill.isAnException(url) && emailManager.isSignedIn())) {
-                webView.evaluateJavascript("javascript:${javaScriptInjector.getFunctionsJS()}", null)
-            }
+            // if (isDuckDuckGoUrl(url) || (isFeatureEnabled() && !autofill.isAnException(url) && emailManager.isSignedIn())) {
```

Comment:
This guard needs disabled for now, but we likely need to make changes here based on the conversations earlier today. Leaving the comments in (messily) as a reminder until the next clean up